internal/cli/version.go

```diff
@@ -67,6 +70,7 @@ func NewCmdVersion() *cobra.Command {
 
 func (o *VersionOptions) Bind(fs *pflag.FlagSet) {
 	fs.StringVarP(&o.Output, "output", "o", o.Output, fmt.Sprintf("Output format. One of: (%s).", strings.Join(legalVersionOutputTypes, ", ")))
+	fs.DurationVar(&o.Timeout, "timeout", o.Timeout, "Timeout for version request")
```

Comment:
"timeout" is a bit non-descriptive. Timeout of what? How about "request-timeout"?

Reply:
removed